### ISSUE TITLE
Accessibility issues with screen reader

### ISSUE THREAD

**Issue by froston** [wwayne/react-tooltip#691]
I'm having issues with the `react-tooltip` accessibility with screen readers. The library renders the tooltip structure in this way:

![image](https://user-images.githubusercontent.com/24678624/113428410-225bd100-93ad-11eb-9680-f7f7be876e7c.png)

That causes the screen reader to read all the style tag content before getting to the tooltip's text, which is not desired. 
I guess the style tag should either be moved outside the tooltip's markup or contain `aria-hidden="true"` attribute in order to be skipped by screen readers. 
Is there some other solution to this issue? Thank you.

**Comment by dearamerican** [comment]
Ah, actually I've got it sorted--thanks anyway! For folks coming to this thread trying to make accessible tooltips, here's what worked for me:
       ```
 <ReactTooltip
          id={id}
           // ... (other props) ...
          event="click"
          role="status"
        />
        <button
          data-tip
          aria-controls={id}
        >
          {children}
        </button>
```

Use `event="click"` on ReactTooltip (so activate on click, not on hover), because otherwise keyboard users cannot trigger the tooltip when it is focused.

Adding `aria-controls={id}` gets the info announced by screen readers on trigger.

**Comment by daceynolan** [comment]
@dearamerican I tried this solution, but it will keep repeating the tooltip message. Did you have this same problem?
